packages/contracts-bedrock/contracts/dispute/interfaces/IDisputeGameFactory.sol

```diff
@@ -5,105 +5,87 @@ import "../../libraries/DisputeTypes.sol";
+    /// @notice `gameImpls` is a mapping that maps `GameType`s to their respective
+    ///         `IDisputeGame` implementations.
+    /// @param gameType The type of the dispute game.
+    /// @return _impl The address of the implementation of the game type.
+    ///         Will be cloned on creation of a new dispute game with the given `gameType`.
     function gameImpls(GameType gameType) external view returns (IDisputeGame _impl);
 
-    /**
-     * @notice Creates a new DisputeGame proxy contract.
-     * @param gameType The type of the DisputeGame - used to decide the proxy implementation
-     * @param rootClaim The root claim of the DisputeGame.
-     * @param extraData Any extra data that should be provided to the created dispute game.
-     * @return proxy The address of the created DisputeGame proxy.
-     */
+    /// @notice Creates a new DisputeGame proxy contract.
+    /// @param gameType The type of the DisputeGame - used to decide the proxy implementation
+    /// @param rootClaim The root claim of the DisputeGame.
+    /// @param extraData Any extra data that should be provided to the created dispute game.
+    /// @return proxy The address of the created DisputeGame proxy.
     function create(
         GameType gameType,
         Claim rootClaim,
         bytes calldata extraData
     ) external returns (IDisputeGame proxy);
 
-    /**
-     * @notice Sets the implementation contract for a specific `GameType`.
-     * @dev May only be called by the `owner`.
-     * @param gameType The type of the DisputeGame.
-     * @param impl The implementation contract for the given `GameType`.
-     */
+    ///
+    /// @notice Sets the implementation contract for a specific `GameType`.
+    /// @dev May only be called by the `owner`.
+    /// @param gameType The type of the DisputeGame.
+    /// @param impl The implementation contract for the given `GameType`.
+    ///
     function setImplementation(GameType gameType, IDisputeGame impl) external;
 
-    /**
-     * @notice Returns a unique identifier for the given dispute game parameters.
-     * @dev Hashes the concatenation of `gameType . rootClaim . extraData`
-     *      without expanding memory.
-     * @param gameType The type of the DisputeGame.
-     * @param rootClaim The root claim of the DisputeGame.
-     * @param extraData Any extra data that should be provided to the created dispute game.
-     * @return _uuid The unique identifier for the given dispute game parameters.
-     */
+    /// @notice Returns a unique identifier for the given dispute game parameters.
+    /// @dev Hashes the concatenation of `gameType . rootClaim . extraData`
+    ///      without expanding memory.
+    /// @param gameType The type of the DisputeGame.
+    /// @param rootClaim The root claim of the DisputeGame.
+    /// @param extraData Any extra data that should be provided to the created dispute game.
+    /// @return _uuid The unique identifier for the given dispute game parameters.
```

Comment:
Maybe rip into another pr so we don't have to regen gas snapshot, bindings, etc etc, but would be nice to use the underscore postfix return standard

Reply:
For sure, can do that in a separate PR.